### ISSUE TITLE
No focus states for Categories - a11y issue

### ISSUE THREAD

**Issue by melodyma** [veteransaffairscanada/vac-benefits-directory#737]
When using keyboard tabbing into Categories, cannot tell that you're focusing on any categories because there are no focus states for them.

**Comment by melodyma** [comment]
Thanks for fixing this @maxneuvians! Found a design accessibility issue on this as outcome. Opened new issue #797

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility work](https://trello.com/c/H7Q1Ycrm/93-accessibility-work)
